src/middleware.js

```diff
@@ -0,0 +1,19 @@
+// middleware.js
+import express from 'express';
+import morgan from 'morgan';
+import helmet from 'helmet';
+
+export function setupMiddleware(app) {
+  // Apply middleware
+//   app.use(express.json());
+//   app.use(morgan('combined'));
+//   app.use(helmet());
+
+  // Custom middleware for logging
+  app.use((req, res, next) => {
```

Comment:
Is this logic duplicated in the `index.js` file?

Reply:
yes it has been removed now, the middleware config has issues, I will figure it out, once i do I should be able to completey seperate the concerns, i.e putting the last post requests int he route file